### PR TITLE
Skift remove jquery

### LINKED ISSUE #10: Drop jQuery dependency
We want to drop jQuery dependency.

This one is blocked by #34

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

Fixes #10

### CODE REVIEW

**Comment by nealstewart** [comment]
@thomasthiebaud Hello hello!

This is a large and sweeping change. Can you document the changes that you've made in some kind of change log?

Which APIs are removed/changed?

**Comment by nealstewart** [comment]
See here:

https://github.com/trustpilot/skift/blob/master/CHANGELOG.md

**Comment by nealstewart** [comment]
The repository was using some kind of auto-generated CHANGELOG, but no one left in the company seems to be sold on that, so it's just the manual one for now :)

**Comment by thomasthiebaud** [comment]
Sure I can do that

File: examples/serve.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
-const http = require('http');
```
[the remark this answers is not in the document]
Comment (thomasthiebaud) [reply examples/serve.js]: I used `webpack-serve` instead, which allow you to have hot reloading when developing. It's much more convenient.

File: types/main.d.ts [hunk 2]
```diff
@@ -16,7 +15,7 @@ export declare const tests: SplitTest[];
 export declare const testsObservable: BehavioralSubject<SplitTest[]>;
 export declare function config(userConfig?: UserConfig): void;
 export declare function initialize(): void;
-export declare function getUserAgentInfo(): UserAgentInfo;
+export declare function getUserAgentInfo(): import("../../../../../Users/tht/Trustpilot/skift/src/useragentinfo").UserAgentInfo;
```
[the remark this answers is not in the document]
Comment (thomasthiebaud) [reply types/main.d.ts]: No indeed, I will remove it

**Comment by thomasthiebaud** [comment]
I'm closing this PR and will open smaller PRs

**Pull request closed without merging**